services/blockchain-indexer/tests/unit/shared/dataservice/business/mainchain.test.js

```diff
@@ -0,0 +1,172 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../../../../config');
+
+const mockedFilePath = '../../../../../shared/dataService/business/network';
+
+beforeEach(() => jest.resetModules());
+
+describe('Test isMainchain method', () => {
+	jest.mock('lisk-service-framework', () => {
+		const actual = jest.requireActual('lisk-service-framework');
+		return {
+			...actual,
+			CacheRedis: jest.fn(),
+			CacheLRU: jest.fn(),
+		};
+	});
+
+	it('should return false -> undefined chainID', async () => {
```

Comment:
Nice way of defining test description 👍 But would it be better to use `should...when...` for the sake of consistency? same for other tests where applicable.

```suggestion
	it('should return false -> undefined chainID', async () => {
```